platforms/netlify/README.md

```diff
@@ -1,8 +1,8 @@
 # Introduction
 
-Prisma and Netlify (current build bot) integration.
+Prisma and Netlify (current build CI) integration.
 
 # Friction Points
 
-- Since Netlify's current build bot doesn't use the newer version of `zip-it-and-ship-it`, we need to bundle the binaries ourselves. See `build.sh`.
+- Since Netlify's current build CI doesn't use the newer version of `zip-it-and-ship-it`, we need to bundle the binaries ourselves. See `build.sh`.
```

Comment:
`zip-it-and-ship-it` could be linked to the repo

Reply:
For future!